scripts/deploy.js

```diff
@@ -360,6 +360,17 @@ async function ensureVerifierDeployment(plan, config) {
     }
 }
 
+async function ensureStateBridgeDeployment(plan, config) {
+    if (!config.stateBridgeContractAddress) {
+        config.stateBridgeContractAddress = process.env.STATE_BRIDGE_CONTRACT_ADDRESS;
+    }
+    if (!config.stateBridgeContractAddress) {
+        config.stateBridgeContractAddress = await ask(
+            'Enter state bridge contract address: '
+        );
+    }
```

Comment:
If the user doesn't provide the parameter, the address is `undefined` and things will break. Instead, if the user's response is empty, it should default to not having a state bridge address (passing `0` as the address). See [`ensureTreeDepth`](https://github.com/worldcoin/world-id-contracts/blob/main/scripts/deploy.js#L184) for an example of how to do this in a user-friendly fashion.

Reply:
fixed